### ISSUE TITLE
Commit status should be shown on the commit page

### ISSUE THREAD

**Issue by youcandanch** [isaacs/github#120]
The commit status API is awesome. We're digging using it for pull requests and on the branch list page - however, I would love to see it handled right on the commit page (just for an example, https://github.com/jgeurts/FluentScheduler/commit/b7b24b666c8da5c27a241f4383a3922ebd21f208). I think there'd really be some benefit to having it right on that page, especially since it's available via the API already.

(Unless I'm totally missing something and it is, in which case, I'm silly and my bad.)

**Comment by mmun** [comment]
I agree. Kinda surprised it's not there, since build status is scoped to a commit.
